.github/workflows/runbenchandupload.yml

```diff
@@ -0,0 +1,62 @@
+name: RunUploadBenchmarks
+env:
+  JULIA_NUM_THREADS: 1
```

Comment:
do we really want that for multithreaded benchmarks?

Reply:
Multithreaded benchmarks are quite poor right now, but if in the future are more developed we can either have a seperate job for them or increase the threads here.
For consistency we should always benchmark serial code with a single thread, so I figured we might as well declare we need a single thread.
However, I was actually been thinking that we might as well use 2 threads just for the parallel precompilation/instantiation which I think Julia supports since a recent version. So, with this mindset, I would then change it back to `JULIA_NUM_THREADS: 2`.

Reply:
we might as well set the number of threads to `"auto"` (these days, GitHub runners have 4)

Reply:
ok